### DISCUSSION TITLE
Verification failure

### DISCUSSION THREAD

**Discussion by twretro** [Freetz-NG/freetz-ng#647] [Allgemein]
English: Hello! 
I try to create my first Freetz-NG Firmware for a Fritzbox 7560 VF but I always get an Error:

Deutsch: Hallo zusammen! Ich versuche grade meine erste Freetz-NG Firmware für eine Fritzbox 7560 VF zu erstellen, bekomme aber jedes Mal folgenden Fehler:
Kann mir jemand sagen, wie ich das beheben kann?

STEP 1: UNPACK
source firmware file: dl/fw/FRITZ.Box_7560-07.29.image
checking signature: failed
OpenSSL: 4027CF18617F0000:error:02000068:rsa routines:eek:ssl_rsa_verify:bad signature:../crypto/rsa/rsa_sign.c:430:
4027CF18617F0000:error:1C880004:provider routines:rsa_verify:RSA lib:../providers/implementations/signature/rsa_sig.c:774:
Verification failure
ERROR: image could not be verified with public key
FINISHED

How can I fix this?

**Comment by fda77** [comment]
Entweder die Überprung abschalten (blöd) oder ein Image von AVM nehmen
